src/scss/custom/_stepper.scss

```diff
@@ -0,0 +1,36 @@
+.stepper__step {
+  color: $primary;
+  font-weight: 500;
+  &__value {
+    color: $primary;
+    border-color: $primary !important;
```

Comment:
Recently I've been leaning more towards using Bootstrap classes than any custom CSS (as much as possible).  At the moment there is a bit of a mix between the two - this border could be done with a `border-primary` class for instance.